packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -211,22 +228,37 @@ contract Multisig is PluginUUPSUpgradeable, Addresslist {
         bytes calldata _metadata,
         IDAO.Action[] calldata _actions,
         bool _approveProposal,
-        bool _tryExecution
+        bool _tryExecution,
+        uint64 _startDate,
+        uint64 _endDate
     ) external returns (uint256 proposalId) {
         uint64 snapshotBlock = block.number.toUint64() - 1;
 
         if (multisigSettings.onlyListed && !isListedAtBlock(_msgSender(), snapshotBlock)) {
             revert ProposalCreationForbidden(_msgSender());
         }
 
+        if (_startDate == 0) {
+            _startDate = block.timestamp.toUint64();
+        }
+
+        if (_startDate < block.timestamp.toUint64()) {
+            revert InvalidStartDate({limit: block.timestamp.toUint64(), actual: _startDate});
+        }
+
+        if (_endDate < _startDate) {
+            revert InvalidEndDate({limit: _startDate, actual: _endDate});
+        }
+
         proposalId = proposalCounter.current();
 
         // Create the proposal
         Proposal storage proposal_ = proposals[proposalId];
 
-        proposal_.open = true;
         proposal_.parameters.snapshotBlock = snapshotBlock;
         proposal_.parameters.minApprovals = multisigSettings.minApprovals;
+        proposal_.parameters.startDate = _startDate;
+        proposal_.parameters.endDate = _endDate;
         proposal_.tally.addresslistLength = addresslistLengthAtBlock(snapshotBlock); // TODO https://aragonassociation.atlassian.net/browse/APP-1417
```

Comment:
The // TODO can be removed as https://aragonassociation.atlassian.net/browse/APP-1417 was closed. Please also remove it from `AddresslistVoting`.

Reply:
done